examples/benchmark/main.zig

```diff
@@ -5,11 +5,23 @@ const flags = @import("tigerbeetle/flags");
 
 const async_ = asynk.async_;
 
+// set this to false to disable the verbose logging
+const show_mlir = true;
+
+pub const std_options = .{
+    .log_level = .warn,
+    .log_scope_levels = &[_]std.log.ScopeLevel{
+        .{ .scope = .pjrt, .level = if (show_mlir) .debug else .err },
```

Comment:
I really don't like this, because it's misleading. 
Using `show_mlir` should not change the logging level of `pjrt`.
And `.err` is **not** a good default for logging, it should be to `.warn` if we are spamming the `warm` level, let's fix that instead.

For this simple benchmark code, I think setting `.log_level` to debug to see the IR is enough.

Reply:
Sure!
```zig 
pub const std_options = .{
    .log_level = .debug,
};
```
do the job for this example